ibm/resource_ibm_is_lb.go

```diff
@@ -501,6 +508,16 @@ func lbGet(d *schema.ResourceData, meta interface{}, id string) error {
 		if profile.Name != nil {
 			d.Set(isLBProfile, *lb.Profile.Name)
 		}
+	} else {
+		if lb.Logging != nil {
```

Comment:
here you can have only 1 if condition if lb.logging != nil && lb.logging.Datapath != nil &&l b.logging.Datapath.Active != nil

Reply:
@hkantare Changes Incorporated